pkg/gds/members.go

```diff
@@ -197,7 +197,11 @@ func (s *Members) List(ctx context.Context, in *api.ListRequest) (out *api.ListR
 		}
 
 		// Collect the VASP from the iterator
-		vasp := iter.VASP()
+		var vasp *pb.VASP
+		if vasp, err = iter.VASP(); err != nil {
+			log.Warn().Err(err).Msg("could not parse VASP from database")
```

Comment:
In certs.go you used `log.Error()` and here `log.Warn()` - any reason for the difference?

Reply:
Good point, I think I intended this to be a `log.Error`.